plugin/trino-iceberg/src/test/resources/jdbc/create-table.sql

```diff
@@ -0,0 +1,20 @@
+/**
+  Table definition in Iceberg repository:
+  https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java
```

Comment:
How do we make sure this file stays in sync with the referenced one?

or, maybe we can avoid having a copy here?

Reply:
It looks like the namespace_properties_table changes along with others we were waiting for just got released on 0.14.0 a few days ago.

Maybe we could now reference the static variable [JdbcUtil.CREATE_NAMESPACE_PROPERTIES_TABLE](https://github.com/apache/iceberg/blob/6d2edd6284ebc5301dbe45376a31ca8316852a77/core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java#L86) here going forward

Reply:
> It looks like the namespace_properties_table changes along with others

what are the changes?
does it mean we should test this catalog against Iceberg 13 and Iceberg 14 as well?

Reply:
They changed column names in https://github.com/apache/iceberg/pull/5017. The PR doesn't contain logic to handle the old table definition, so I suppose they just broke the backward compatibility. 

> does it mean we should test this catalog against Iceberg 13 and Iceberg 14 as well?

I think so. (or clarify the supported version in documentation)

Reply:
> They changed column names in [apache/iceberg#5017](https://github.com/apache/iceberg/pull/5017). The PR doesn't contain logic to handle the old table definition, so I suppose they just broke the backward compatibility.

Concerning

Reply:
According to this issue on Iceberg repo no backwards compatible was broken because the previous version of this was never released? https://github.com/apache/iceberg/issues/4952

Reply:
In fact I can confirm the Iceberg guys didn't release a breaking change, see the file in question at 0.13.2 https://github.com/apache/iceberg/blob/apache-iceberg-0.13.2/core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java where this `NAMESPACE_PROPERTY_KEY` didn't exist, and now see same file at 0.14.0, where it does exist with the new value https://github.com/apache/iceberg/blob/apache-iceberg-0.14.0/core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java#L83.

So sounds like it was never released with the old value

Reply:
@nfcampos Your confirmation looks correct. Do you know if they have compatibility test in the repository?

Reply:
Not that I know of, no